src/dev/getting_started.md

```diff
@@ -0,0 +1,100 @@
+# Building and Running Tests
+
+## Repos
+
+Yorick is spread over a handful of repos, but the two you are likely to need to clone are:
+
+ - [ykrustc](https://github.com/softdevteam/ykrustc): The compiler.
+ - [yk](https://github.com/softdevteam/ykrustc): The runtime parts of the system.
+
+The latter is a monorepo containing a few different crates.
+
+## Building and Testing the Compiler
+
+### Building the Compiler
+
+To build the compiler, run:
+```
+./x.py build --stage 1
+```
+
+A stage one build is usually sufficient for development.
+
+If you have made changes to `ykpack` (in the `yk` repo), then edit the
+`Cargo.toml` in the top level of `ykrustc` and uncomment (and adapt if needed)
+the lines:
+
+```
+#[patch."https://github.com/softdevteam/yk"]
+#ykpack = { path = "../yk/ykpack" }
+```
+
+This will override the compiler's dependency to your local version.
+
+### Testing the Compiler
+
+The compiler is tested with:
+```
+./x.py test --stage 1
+```
+
+You can do more thorough testing by testing stage 2, but it takes quite a
+while. CI will always to a full stage 2 test.
```

Comment:
"to" -> "do"?

Reply:
Fixed in 8d65940ecc47bbac5cbc0fc8f75f2e219a6f5850